packages/calcite-components/src/components/card/card.scss

```diff
@@ -52,35 +103,49 @@
 }
 
 .card-content {
-  @apply text-color-3
-    text-n2-wrap
-    p-3;
+  @apply h-auto text-n2-wrap;
+}
+
+.has-slotted-content {
+  @apply p-3;
 }
 
-:host([selected]) .calcite-card-container {
-  @apply border-color-brand;
+:host([selected]) .content-wrapper {
+  box-shadow: inset 0 -4px 0 0 var(--calcite-card-accent-color-selected);
 }
 
-// slotted content
+// to be removed in future release
```

Comment:
Could we link to the PR/issue to give more context when it's time to remove?

Reply:
Yeah, do we want a running list of “deprecated items to be removed next major release” (for all components), in an issue or somewhere else?

Reply:
Removed the comments for now.